app/scripts/views/ready.js

```diff
@@ -102,6 +110,17 @@ define(function (require, exports, module) {
       const readyToSyncText = this._templateInfo.readyToSyncText;
       // translateInTemplate HTML escapes
       return this.translateInTemplate(readyToSyncText);
+    },
+
+    /**
+     * Get the HTML escaped "Ready" text.
+     *
+     * @returns {String}
+     */
+    _getEscapedReadyText () {
+      const readyText = this._templateInfo.readyText;
```

Comment:
Not all types delcare a `readyText`, can that cause problems?

Reply:
Update this name, `readyText` is actually `emailReadyText` since only renders for emails.